pkg/controller/download_request_controller_test.go

```diff
@@ -18,310 +18,256 @@ package controller
+	"k8s.io/client-go/kubernetes/scheme"
 
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	"github.com/vmware-tanzu/velero/pkg/builder"
-	"github.com/vmware-tanzu/velero/pkg/generated/clientset/versioned/fake"
-	informers "github.com/vmware-tanzu/velero/pkg/generated/informers/externalversions"
 	"github.com/vmware-tanzu/velero/pkg/persistence"
 	persistencemocks "github.com/vmware-tanzu/velero/pkg/persistence/mocks"
 	"github.com/vmware-tanzu/velero/pkg/plugin/clientmgmt"
 	pluginmocks "github.com/vmware-tanzu/velero/pkg/plugin/mocks"
 	velerotest "github.com/vmware-tanzu/velero/pkg/test"
-	kubeutil "github.com/vmware-tanzu/velero/pkg/util/kube"
 )
 
-type downloadRequestTestHarness struct {
-	client          *fake.Clientset
-	informerFactory informers.SharedInformerFactory
-	pluginManager   *pluginmocks.Manager
-	backupStore     *persistencemocks.BackupStore
-
-	controller *downloadRequestController
-}
-
-func newDownloadRequestTestHarness(t *testing.T, fakeClient client.Client) *downloadRequestTestHarness {
-	var (
-		client          = fake.NewSimpleClientset()
-		informerFactory = informers.NewSharedInformerFactory(client, 0)
-		pluginManager   = new(pluginmocks.Manager)
-		backupStore     = new(persistencemocks.BackupStore)
-		controller      = NewDownloadRequestController(
-			client.VeleroV1(),
-			informerFactory.Velero().V1().DownloadRequests(),
-			informerFactory.Velero().V1().Restores().Lister(),
-			fakeClient,
-			informerFactory.Velero().V1().Backups().Lister(),
-			func(logrus.FieldLogger) clientmgmt.Manager { return pluginManager },
-			velerotest.NewLogger(),
-		).(*downloadRequestController)
-	)
-
-	clockTime, err := time.Parse(time.RFC1123, time.RFC1123)
-	require.NoError(t, err)
-	controller.clock = clock.NewFakeClock(clockTime)
-
-	controller.newBackupStore = func(*velerov1api.BackupStorageLocation, persistence.ObjectStoreGetter, logrus.FieldLogger) (persistence.BackupStore, error) {
-		return backupStore, nil
-	}
-
-	pluginManager.On("CleanupClients").Return()
-
-	return &downloadRequestTestHarness{
-		client:          client,
-		informerFactory: informerFactory,
-		pluginManager:   pluginManager,
-		backupStore:     backupStore,
-		controller:      controller,
-	}
-}
-
-func newDownloadRequest(phase velerov1api.DownloadRequestPhase, targetKind velerov1api.DownloadTargetKind, targetName string) *velerov1api.DownloadRequest {
-	return &velerov1api.DownloadRequest{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "a-download-request",
-			Namespace: velerov1api.DefaultNamespace,
-		},
-		Spec: velerov1api.DownloadRequestSpec{
-			Target: velerov1api.DownloadTarget{
-				Kind: targetKind,
-				Name: targetName,
+var _ = Describe("Download Request Reconciler", func() {
+	BeforeEach(func() {})
+	AfterEach(func() {})
```

Comment:
Can these be removed if we're not doing anything in setup/teardown?

Reply:
Yes, good catch, thank you! I'll remove it from #3180 too as well as for other merged controllers.